keps/sig-network/563-dual-stack/README.md

```diff
@@ -1477,16 +1516,10 @@ resource usage (CPU, RAM, disk, IO, ...) in any components?**
 
 ### Troubleshooting
 
-The Troubleshooting section currently serves the `Playbook` role. We may consider
-splitting it into a dedicated `Playbook` document (potentially with some monitoring
-details). For now, we leave it here.
-
-
 * **How does this feature react if the API server and/or etcd is unavailable?**
   This feature will not be operable if either kube-apiserver or etcd is unavailable.
 
 * **What are other known failure modes?**
-  For each of them, fill in the following information by copying the below template:
```

Comment:
I think this doesn't address workload failures nor the "imbalance", we probably want to discuss those.